hack/verify-flags/known-flags.txt

```diff
@@ -187,6 +187,8 @@ exit-on-lock-contention
 experimental-allowed-unsafe-sysctls
 experimental-bootstrap-kubeconfig
 experimental-keystone-url
+experimental-mounter-path
```

Comment:
Removing this line will reduce your potential rebase burden, as this PR still relies on mounter-path and you already added experimental-mounter-path to known flags in #35646